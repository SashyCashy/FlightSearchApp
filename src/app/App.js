import React from 'react';
import Navbar from '../components/common/Navbar';
import Content from '../components/layout';
import './App.css';

function App() {
  return (
    <div className="App">
      <Navbar />
      <Content />
    </div>
  );
}

export default App;
