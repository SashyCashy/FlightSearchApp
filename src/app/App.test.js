import React from 'react';

import { shallow } from 'enzyme';
import App from './App';
describe('<App /> ', () => {
  it('Check the App structure ', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
