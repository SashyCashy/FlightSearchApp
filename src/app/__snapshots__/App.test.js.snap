// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App />  Check the App structure  1`] = `
<div
  className="App"
>
  <Navbar
    navbarTitle="Customer Details"
  />
  <Content />
</div>
`;
